Stage 2

stino-x opened the pull request:
a little bit funny looking at any screen size after 768 px but that was in the knowledge that next week I will put a media query for the laptop screen sizes and beyond this was just to cater to tabs and mobile phones especially, as the challenge said

stino-x committed:
added image depot

adding the file that stores the images

stino-x committed:
adding images

adding images

stino-x committed:
adding all necessary code

this is the code meant to complete the task

stino-x committed:
completing the task

i added the necessary HTML and CSS to the project to complete it

stino-x committed:
i added a nav tag

i gave the hamburger a nav tag

emmiiorji commented on styles.css:
```diff
@@ -0,0 +1,343 @@
+  backdrop-filter: blur(0.5px);
+}
+
+.section-2 {
+  margin-bottom: 3rem;
+}
```
You have a good design. However, as mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/100a933087f033def9233c8cce29c72194391fe0/portfolio/2_mobile.md?plain=1#L51), kindly use a `display: grid` property for the container of the works section.

emmiiorji commented on index.html:
```diff
@@ -0,0 +1,195 @@
+                </div>
+            </div>
+        </section>
+        <section class="section-3">
+            <h1 class="about-sign">About</h1>
+             <p>
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ab culpa iste nam qui excepturi facere at quis porro odio nulla in quod dignissimos, sint cumque quo! Repellat ipsum nulla a.
+             </p>
+             <button class="Start-collab">
+                Get my CV
+                <img src="images.svg\ic_download (2).svg" alt="">
+             </button>
+             <div class="color-boxes">
+                <div class="blue">
+                    <h3>Languages</h3>
+                    <ul>
+                        <li>Javascript(ES5/ES6)</li>
```
There seems to be some horizontal scroll due to some elements overflowing. Kindly adjust this for a better user experience on the website. This will also help to centralize some elements that are not currently centralized.

![image](https://user-images.githubusercontent.com/20554583/205469480-9be3dd85-4094-4d2f-8343-e6e6f538cfe5.png)

stino-x replied:
how do i do this please?

emmiiorji commented on index.html:
```diff
@@ -0,0 +1,195 @@
+                    </button>
+                </div>
+            </div>
+            <div class="project-items">
+                <img src="images.svg\Rectangle 21 (19).svg" class="project-image" alt="">
+                <div class="languages">
+                    <h2>Project name goes here</h2>
+                    <ul class="H-R-J">
+                        <li class="Html-css">HTML/CSS</li>
+                        <li class="Ruby">RUBY ON RAILS</li>
+                        <li class="Javascript">JAVASCRIPT</li>
+                    </ul>
```
As mentioned in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/100a933087f033def9233c8cce29c72194391fe0/portfolio/2_mobile.md?plain=1#L51), kindly use a display: flex property for the container of each listed project.

Here's it's style currently:
![image](https://user-images.githubusercontent.com/20554583/205469579-ec50ead2-f2ee-49ef-8115-01c838db18ad.png)

stino-x replied:
hey bro, i used flex there I can show you a screenshot, there's no way I could have positioned that thing without that outside of using position relative which i didn't use

emmiiorji replied:
Hi @stino-x

Thanks for your response. You did not seem to have a style defined for the project-items class., hence, the use of the default style for a div element. Kindly confirm this.

stino-x replied:
https://drive.google.com/file/d/1O8oCHWOWtt8QC9TVqRT41kusbAecbTK5/view?usp=sharing

that is a screen-shot of my css via my drive

stino-x replied:
> Hi @stino-x
> 
> Thanks for your response. You did not seem to have a style defined for the project-items class., hence, the use of the default style for a div element. Kindly confirm this.

i could do a video as well if you want

stino-x replied:
> Hi @stino-x
> 
> Thanks for your response. You did not seem to have a style defined for the project-items class., hence, the use of the default style for a div element. Kindly confirm this.

a video showing what happens when i take it out:
https://drive.google.com/file/d/1k7UOi_vGn21cpk-qb3MjV8fKgh46fYiF/view?usp=sharing

emmiiorji replied:
Hi @stino-x,

Thank you for the very careful illustrations. Obviously you have put in a lot of work in the project and deserve to be rewarded for that, therefore I can see your project has been approved.

To be clearer, the `H-R-J` class as you have it styles the `ul` element containing the technologies listing. However, the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/100a933087f033def9233c8cce29c72194391fe0/portfolio/2_mobile.md?plain=1#L51) stated that the individual cards of the works section be laid out using css flexbox. This means that the container having all the content of a particular project card which in your case is `project-items` should have a `display: flex` property.

Sticking to requirements as this will help to strengthen your use of the tools that constitute the learning objectives.

I hope this helps to clarify. Once again, congratulations and happy coding👏👏👏.

emmiiorji reviewed (changes requested):
Hi @stino-x, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![A little bit more](https://user-images.githubusercontent.com/20554583/205469312-655f27ca-8499-44d5-8488-6955c09699b2.gif)

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

stino-x committed:
All necessary changes made.

i removed the scrolling issue and also checked the flex for my lists and i had in all of them i can verify with screenshots

roshan-bajgain reviewed (approved):
Hi @stino-x, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

stino-x committed:
completing form

i added the rest of the form according to what the challenge says

emmiiorji reviewed (changes requested):
Hi @stino-x, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

![Invalid2](https://user-images.githubusercontent.com/20554583/205705654-06025ad5-9c66-4997-b9db-f595213b7ee4.gif)

You have submitted an old pull request for a new milestone. Kindly note that each milestone requires making a new feature branch from the main (the branch name should be descriptive), implementing the requirements and creating a new pull request from the feature branch to the main branch.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
